doc/glossary/index.rst

```diff
@@ -31,3 +31,374 @@ Glossary
+
+   Time Window
+        Timewindow in which the given OTP value is valid for authentication.
+        ``Timestep and timewindow`` are completely similar to the ``counter and countwindow`` (:ref:`tokeninfo`) of HOTP tokens.
+
+   Revoked Token
+        Tokens can be ``revoked``. Usually this means the token is ``disabled and locked``. A locked token can
+        not be modified anymore. It can only be deleted. Certain token types like certificate may
+        define special actions when revoking a token.
+
+   Orphaned Token
+        An orphaned token means, that it has a user assigned, but the user does not exist in the
+        user store (anymore).
+
+   Disabled Token
+        Tokens can be ``disabled``. Disabled tokens still belong to the assigned user but those tokens
+        can not be used to authenticate. ``Disabled tokens can be enabled again``.
+
+   Resolver(UserId)
+        UserIdResolvers are connectors to those user stores, the locations,
+        where the users are managed. Nowadays this can be LDAP directories or
+        especially Active Directory, some times FreeIPA or the Redhat 389 service.
+        But classically users are also located in files like ``/etc/passwd`` on
+        standalone unix systems. Web services often use SQL databases as
+        user store.
+
+        Today with many more online cloud services SCIM is also an uprising
+        protocol to access userstores.
+
+        privacyIDEA already comes with :ref:`useridresolvers` to talk to all these
+        user stores:
+
+        * :ref:`flatfile_resolver`
+        * :ref:`ldap_resolver`
+        * :ref:`sql_resolver`
+        * :ref:`scim_resolver`
+        * :ref:`http_resolver`
+
+   Resolver(Machine)
+        Machine Resolvers are used to find machines in directories like LDAP,
+        Active Directory, puppet, salt, or the /etc/hosts file.
+
+        The idea is for users to be able to authenticate on those client machines.
+        Not in all cases an online authentication request is possible,
+        so that authentication items can be passed to those client machines.
+
+        In addition you need to define, which application on the client machine
+        the user should authenticate to.
+        Different application require different authentication items.
+
+        Therefore privacyIDEA can define application types.
+        At the moment privacyIDEA knows the application
+        ``luks``, ``offline`` and ``ssh``.
+
+   Tokeninfo
+        The table “tokeninfo” is used to store additional, long information that is
+        specific to the :ref:`tokentypes`. E.g. the tokentype “TOTP” has additional entries
+        in the tokeninfo table for “timeStep” and “timeWindow”, which are stored in the
+        column “Key” and “Value”.
+
+        The ``tokeninfo`` is reference by the foreign key to the “token” table.
+
+        Token info can be viewed and partially edited in the GUI. In addition, the
```

Comment:
GUI -> WebUI (consistent naming)

Reply:
done in f61a46945fb583cac2e66ad33600bb26eae87ded.